palace.pyx

```diff
@@ -374,6 +376,58 @@ cdef class Context:
         self.impl.update()
 
 
+cdef class Listener:
+    cdef alure.Listener impl
+
+    def __init__(self, context: Context) -> None:
+        self.impl = context.impl.get_listener()
+
+    def __bool__(self) -> bool:
+        return <boolean> self.impl
+
+    def set_gain(self, value: float) -> None:
+        self.impl.set_gain(value)
+
+    def set_position(self, value: Vector3) -> None:
+        self.impl.set_position(to_vector3(value))
+
+    def set_velocity(self, value: Vector3) -> None:
+        self.impl.set_velocity(to_vector3(value))
+
+    def set_orientation(self, value: Tuple[Vector3, Vector3]) -> None:
+        at, up = value
+        self.impl.set_orientation(
+            pair[alure.Vector3, alure.Vector3](to_vector3(at), to_vector3(up)))
+
+    def set_meters_per_unit(self, value: float) -> None:
+        self.impl.set_meters_per_unit(value)    
+
+    gain = property(fset=set_gain, doc='Master gain for all context output.')
+    position = property(fset=set_position, doc='3D position of the listener.')
+    velocity = property(
+        fset=set_velocity
+        doc=(
+            """3D velocity of the listener, in units per second. As with
+            OpenAL, this does not actually alter the listener's position, and
+            instead just alters the pitch as determined by the doppler effect.
+            """))
+    orientation = property(
+        fset=set_orientation
+        doc=(
+            """3D orientation of the listener, using position-relative 'at'
+            and 'up' direction vectors.
```

Comment:
Please use backticks instead ( `` `at` `` and `` `up` ``) in order for Sphinx to generate better doc.